packages/polar/src/lib/account.ts

```diff
@@ -0,0 +1,53 @@
+import { CosmWasmClient } from "secretjs";
+
+import { PolarError } from "../internal/core/errors";
+import { ERRORS } from "../internal/core/errors-list";
+import { Account, PolarRuntimeEnvironment, UserAccount } from "../types";
+import { getClient } from "./client";
+
+export class UserAccountI implements UserAccount {
+  account: Account;
+  client: CosmWasmClient;
+
+  constructor (account: Account, env: PolarRuntimeEnvironment) {
+    this.account = account;
+    this.client = getClient(env.network);
+  }
+
+  async getAccountInfo (): Promise<any> {
+    return await this.client.getAccount(this.account.address);
+  }
+
+  async getBalance (): Promise<any> {
```

Comment:
use a type here

Reply:
Type 'readonly Coin[] | undefined' is not assignable to type 'string'.
Type 'undefined' is not assignable to type 'string'.ts(2322)
@udit-gulati ?